### ISSUE TITLE
Option to remove metadata/assembler directives

### ISSUE THREAD

**Issue by clubby789** [pacak/cargo-show-asm#86]
When decompiling a simple function, the output is something like
```asm
.section .text.poc::func,"ax",@progbits
	.globl	poc::func
	.p2align	4, 0x90
	.type	poc::func,@function
poc::func:

	.cfi_startproc
	mov rax, rdi
	mov byte ptr [rdi], 0
	mov byte ptr [rdi + 2], 0
	mov byte ptr [rdi + 4], 0
	mov byte ptr [rdi + 6], 0
	mov byte ptr [rdi + 8], 0
	mov byte ptr [rdi + 10], 0
	mov byte ptr [rdi + 12], 0
	mov byte ptr [rdi + 14], 0

	ret

.Lfunc_end0:
	.size	poc::func, .Lfunc_end0-poc::func
```

It would be useful to have an option to only output the generated assembly,  (as well as any jump labels) to make comparing code easier - e.g.
```asm
poc::func:
	mov rax, rdi
	mov byte ptr [rdi], 0
	mov byte ptr [rdi + 2], 0
	mov byte ptr [rdi + 4], 0
	mov byte ptr [rdi + 6], 0
	mov byte ptr [rdi + 8], 0
	mov byte ptr [rdi + 10], 0
	mov byte ptr [rdi + 12], 0
	mov byte ptr [rdi + 14], 0
	ret
```

**Comment by pacak** [comment]
Something like this? https://github.com/pacak/cargo-show-asm/pull/87

**Comment by clubby789** [comment]
Looks great, thank you!

**Comment by pacak** [comment]
Released 0.2.4
